src/AzureMcp.csproj

```diff
@@ -77,16 +78,12 @@
     <Message Text="Installing Git hooks..." Importance="high" />
 
     <!-- Run the Install-GitHooks.ps1 script using PowerShell Core -->
-    <Exec Command="pwsh -ExecutionPolicy Bypass -NoProfile -File &quot;$(MSBuildProjectDirectory)/../eng/scripts/Install-GitHooks.ps1&quot;"
-          ContinueOnError="true"
-          ConsoleToMsBuild="true"
-          IgnoreExitCode="true">
-      <Output TaskParameter="ExitCode" PropertyName="ExitCode"/>
+    <Exec Command="pwsh -ExecutionPolicy Bypass -NoProfile -File &quot;$(MSBuildProjectDirectory)/../eng/scripts/Install-GitHooks.ps1&quot;" ContinueOnError="true" ConsoleToMsBuild="true" IgnoreExitCode="true">
+       <Output TaskParameter="ExitCode" PropertyName="ExitCode" />
     </Exec>
 
     <!-- Display informative message if pwsh fails -->
-    <Warning Text="Could not install Git hooks. PowerShell Core (pwsh) is required to install Git hooks:\nhttps://learn.microsoft.com/powershell/scripting/install/installing-powershell"
-             Condition="'$(ExitCode)' != '0'" />
+     <Warning Text="Could not install Git hooks. PowerShell Core (pwsh) is required to install Git hooks:\nhttps://learn.microsoft.com/powershell/scripting/install/installing-powershell" Condition="'$(ExitCode)' != '0'" />
```

Comment:
Is there a diff here?

Reply:
this seems like this PR needs to be rebased on the latest main